app/src/main/java/fr/neamar/kiss/forwarder/ExperienceTweaks.java

```diff
@@ -360,7 +360,11 @@ private boolean isKeyboardOnStartEnabled() {
      * Should the keyboard be displayed?
      */
     private boolean shouldShowKeyboard() {
-        boolean isAssistant = mainActivity.getIntent().getAction().equalsIgnoreCase("android.intent.action.ASSIST");
```

Comment:
can't you just swap the equals order to prevent the NPE?

Reply:
Thanks, it works well